app/helpers/products_helper.rb

```diff
@@ -22,7 +22,7 @@ def measure_test_status_values(tests, task_type)
     [passing, failing, not_started, total]
   end
 
-  def filtering_test_status_values(tests)
+  def filtering_test_status_values(_tests)
```

Comment:
Rubocop did that.

On Friday, January 22, 2016, okeefm <notifications@github.com> wrote:

> In app/helpers/products_helper.rb
> <https://github.com/projectcypress/cypress/pull/309#discussion_r50544682>:
>
> > @@ -22,7 +22,7 @@ def measure_test_status_values(tests, task_type)
> >      [passing, failing, not_started, total]
> >    end
> >
> > -  def filtering_test_status_values(tests)
> > +  def filtering_test_status_values(_tests)
>
> Why'd this get changed to _tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectcypress/cypress/pull/309/files#r50544682>.
>